packages/apollos-api/src/data/content-items/data-source.js

```diff
@@ -16,22 +23,6 @@ class ExtendedContentItem extends ContentItem.dataSource {
     return '';
   };
 
-  _getCursorByParentContentItemId = this.getCursorByParentContentItemId;
-
-  getCursorByParentContentItemId = async (id) => {
-    const cursor = await this._getCursorByParentContentItemId(id);
-
-    return cursor.orderBy('Priority'); // Changed from Core, ordering by Priority instead of 'order'
-  };
-
-  _getCursorBySiblingContentItemId = this.getCursorBySiblingContentItemId;
-
-  getCursorBySiblingContentItemId = async (id) => {
-    const cursor = await this._getCursorBySiblingContentItemId(id);
-
-    return cursor.orderBy('Priority'); // Changed from Core, ordering by Priority instead of 'order'
```

Comment:
@vinnyjth whats' the reason for taking this out?

Reply:
I had patched core to solve this problem - now handled here: 

https://github.com/Differential/willow-creek/pull/125/files#diff-36c37c7a05f5ea49ddc96127943f7f10R12